charts/growthbook/templates/deployment.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "growthbook.fullname" . }}
+  labels:
+    {{- include "growthbook.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "growthbook.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "growthbook.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "growthbook.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: NODE_ENV
+              value: {{ .Values.growthbook.nodeEnv }}
+            - name: MONGODB_URI
+              value: {{ include "growthbook.mongo.uri" . }}
+            - name: JWT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  key: jwt-secret
+                  name: growthbook-secrets
+            - name: ENCRYPTION_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: encryption-key
+                  name: growthbook-secrets
+            - name: APP_ORIGIN
+              value: {{ .Values.growthbook.appOrigin }}
+            - name: API_HOST
+              value: {{ .Values.growthbook.apiHost }}
+            - name: EMAIL_ENABLED 
+              value: "{{ .Values.growthbook.email.enabled }}"
+            {{ if .Values.growthbook.email.enabled }}
+            - name: EMAIL_HOST
+              value: {{ .Values.growthbook.email.host }}
+            - name: EMAIL_PORT
+              value: {{ .Values.growthbook.email.port }}
+            - name: EMAIL_HOST_USER
+              value: {{ .Values.growthbook.email.hostUser }}
+            - name: EMAIL_HOST_PASSWORD
+              value: {{ .Values.growthbook.email.hostPassword }}
+            - name: EMAIL_FROM
+              value: {{ .Values.growthbook.email.sender }}
+            {{ end }}
+          volumeMounts:
+            - name: {{ .Values.volume.name }}
+              mountPath: {{ .Values.volume.mountPath }}
+          ports:
+            - name: frontend-port
+              containerPort: {{ .Values.port.frontendPort }}
```

Comment:
To be honest I don't think there's much value in parameterizing the container ports, since those are anyway internal...
Only the service ports are really worth parameterizing...